rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/LocalDataTransferServer.java

```diff
@@ -245,7 +245,7 @@ public void run()
           try {
             int read = localDataTransferClient.read(dataInfo);
             if (read == -1) {
-              throw new Exception("Could not read data from Non-local node");
+              throw new IOException("Could not read data from Non-local node");
```

Comment:
nit: Make the exception class changes in separate commit

Reply:
done